src/components/stepforms/ConditionsEditor/ConditionsGroup.vue

```diff
@@ -194,6 +192,33 @@ export default class ConditionsGroup extends Vue {
     return groupIndex === this.groups.length - 1;
   }
 
+  async resetOperatorIfNecessary() {
+    // await the re-render to received the updated tree (emitted + received in props)
+    await this.$nextTick();
+
+    // reset the operator to an empty string when there's only one condition left
+    if (this.conditions.length === 1 && this.groups.length === 0) {
+      const newConditionsTree = {
+        ...this.conditionsTree,
+        operator: '',
+      } as AbstractFilterTree;
+      this.emitUpdatedConditionTree(newConditionsTree);
+    }
+    return;
+  }
+
+  setOperatorIfNecessaryAndUpdateConditionTree(newConditionsTree: AbstractFilterTree) {
+    // When there is only one condition and no group, the operator is an empty string (cf resetOperatorIfNecessary).
+    // Set the operator to 'and' when the condition is no longer alone.
```

Comment:
```suggestion
    // Set the operator to 'and' when a group is added.
```

Reply:
hmm, it's not necessary when a group is added, it's when a group or a condition is added, and only in the case it's alone